package.json

```diff
@@ -6,8 +6,10 @@
   "scripts": {
     "prepare": "npm test",
     "test": "eslint .",
+    "test-angular": "eslint ./test/angular/",
```

Comment:
Do we want to run this in our GitHub pipeline?

Reply:
Yes, I had assumed the pipeline just ran `npm test`, but apparently that is not the case. I will add this to the pipeline.